### PR TITLE
NAS-107141 / 12.1 / remove excess logging info when syncing disks

### PR DESCRIPTION
**Pull request by yocalebo**
On large systems, the `Found disks` logging line can cause 100's of lines of information to be printed to `middlewared.log`.

### CODE REVIEW

File: src/middlewared/middlewared/plugins/disk_/sync.py [hunk 1]
```diff
@@ -86,7 +83,6 @@ async def sync_all(self, job):
                 self.logger.warning('Starting disk.sync_all when devd is not connected yet')
 
         sys_disks = await self.middleware.call('device.get_disks')
-        self.logger.info('Found disks: %r', sys_disks)
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/disk_/sync.py:89]: @themylogin  do you remember why we put it there? Should we log at least the names?

Comment (themylogin) [reply src/middlewared/middlewared/plugins/disk_/sync.py:89]: Yes, we have issues when syncing disks not resolved yet, this actually helped me once. @yocalebo can we please leave this line and log only `name`, `ident`, `serial` and `lunid`?

Comment (yocalebo) [reply src/middlewared/middlewared/plugins/disk_/sync.py:89]: Yeah I can do that. But `ident` and `serial` are the same right?

File: src/middlewared/middlewared/plugins/disk_/sync.py [hunk 2]
```diff
@@ -86,7 +83,15 @@ async def sync_all(self, job):
...
+
+        # output logging information to middlewared.log in case we sync disks
+        # when not all the disks have been resolved
+        log_info = {
+            ok: {
+                ik: iv for ik, iv in ov.items() if ik in ('name', 'ident', 'lunid', 'serial')
+            } for ok, ov in sys_disks.items()
+        }
+        self.logger.info(f'Found disks: {log_info}')
```
Comment (themylogin) [inline src/middlewared/middlewared/plugins/disk_/sync.py:94]: Let's stick to old-style formatting for logging please

File: src/middlewared/middlewared/plugins/disk_/sync.py [hunk 3]
```diff
@@ -86,6 +83,14 @@ async def sync_all(self, job):
...
         sys_disks = await self.middleware.call('device.get_disks')
+
+        # output logging information to middlewared.log in case we sync disks
+        # when not all the disks have been resolved
+        log_info = {
+            ok: {
+                ik: iv for ik, iv in ov.items() if ik in ('name', 'ident', 'lunid', 'serial')
+            } for ok, ov in sys_disks.items()
+        }
         self.logger.info('Found disks: %r', sys_disks)
```
Comment (themylogin) [inline src/middlewared/middlewared/plugins/disk_/sync.py:94]: Now yo're logging old variable

Comment (yocalebo) [reply src/middlewared/middlewared/plugins/disk_/sync.py:94]: Typo, fixed.

**Pull request merged**